modules/schema/pages/procedure-lot-part-information.adoc

```diff
@@ -1656,15 +1656,15 @@ shall hold the value associated with that dimension.
 
 The number dimension shall be:
 
-* for weights (BT-5421), elements from the
+* for weights (BT-5421), a value from the
```

Comment:
may be a "code" could be better, "value" could be what is associated to the code

Reply:
Changed as suggested